.github/workflows/production_build.yaml

```diff
@@ -1,6 +1,10 @@
 name: Production build
```

Comment:
@samsonasik It seems I mixed the naming here. Could you update the file name + name to "cache_warmup"? It's definitely not a production build :)

Reply:
implemented https://github.com/rectorphp/getrector.org/pull/388

Reply:
Thanks! :slightly_smiling_face:  Now the confusion makes sense, I've picked very bad naming :/